torch/csrc/jit/serialization/export_bytecode.cpp

```diff
@@ -248,7 +252,54 @@ std::unique_ptr<mobile::Code> compileGraphToMobileCode(
   return mobile_code_ptr;
 }
 
-void checkSchema(const FunctionSchema& schema) {
+std::unique_ptr<mobile::Function> convertJitFunctionToMobileFunction(const GraphFunction *function, const CompilationOptions& options) {
```

Comment:
const GraphFunction&

Reply:
Can you make jitModuleToMobile also calls this function? so we have less code dup

Reply:
sure